app/controllers/requests/Mappings.scala

```diff
@@ -0,0 +1,24 @@
+package controllers.requests
+
+import business.models.User
+import controllers.requests.auth.CreateUserRequest
+import controllers.requests.user.{GetUserResponse, UpdateUserRequest}
+import reactivemongo.bson.BSONObjectID
+
+/**
+ * Created by liviuignat on 22/03/15.
+ */
+object Mappings {

```

Comment:
remove this and put methods closer to where they get used. this can easily become a nightmare to maintain. mind your dependencies.